### PR TITLE
#5232 [Manager] Include license header check for TypeScript in Gradle…

### PR DESCRIPTION
**Pull request by konekoya**
… task

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5232)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -38,6 +38,7 @@ def needHeaders() {
     fileTree(dir: ".").include("**/*.sh"),
     fileTree(dir: ".").include("**/*.properties"),
     fileTree(dir: ".").include("**/*.js"),
+    fileTree(dir: ".").include("**/*.ts"),
```
Comment (chia7712) [inline build.gradle:41]: nice!

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
The fail test is tracked in #5243

**Pull request merged**